theano/tensor/nnet/abstract_conv.py

```diff
@@ -1613,7 +1612,7 @@ def perform(self, node, inp, out_):
         if isinstance(mode, tuple):
             pad = tuple(int(mode[i]) for i in range(self.convdim))
             mode = "valid"
-            new_img = numpy.zeros((img.shape[0], img.shape[1]) +
+            new_img = np.zeros((img.shape[0], img.shape[1]) +
                                   tuple(img.shape[i + 2] + 2 * pad[i]
```

Comment:
[No, it's not](https://www.python.org/dev/peps/pep-0008/#indentation):
> Continuation lines should align wrapped elements either vertically using Python's implicit line joining inside parentheses, brackets and braces, or using a hanging indent

This is not a hanging indent, but a visual indent (i.e., vertically aligned with the content of the opening bracket). `tuple` should start on the same column as `(img`, as both are part of the argument to `np.zeros(`. It was like that before you changed `numpy` to `np`, you need to restore this to pass the code style check!